website/documentation/getting-started/observability/components.md

```diff
@@ -33,14 +33,14 @@ Gardener only provides a monitoring stack if the cluster is not of `purpose: tes
 
 ### Logging into Plutono
 
-Let us start by giving some visual hints on how to access Plutono. [Plutono](https://github.com/credativ/plutono#plutono) allows us to query logs and metrics and visualise those in form of dashboards. Plutono is shipped ready-to-use with a Gardener shoot cluster.
+Let us start by giving some visual hints on how to access Plutono. [Plutono](https://github.com/credativ/plutono#plutono) allows us to query logs and metrics and visualize those in form of dashboards. Plutono is shipped ready-to-use with a Gardener shoot cluster.
 
-In order to access the Gardener provided dashboards, open the `Plutono` link provided in the Gardener dashboard and use the username and password provided next to it.
-
-The password you can use to log in can be retrieved as shown below:
+In order to access the Gardener provided dashboards, open the `Plutono` link provided in the Gardener dashboard. You will be automatically logged in through OIDC based authentication:
 
 ![access-plutono](./images/access-plutono.png)
 
+Programmatic access is still possible via the non-OIDC ingress using the credentials from the `<clustername>.monitoring` secret. It contains the HTTP basic auth credentials in base64-encoded form, as well as the Plutono ingress URL. The Prometheus URL can be derived from the Plutono URL by replacing the `gu` prefix with `p`.
```

Comment:
`programmatic access` although possible is not a publicly supported scenario. We still shall explain that user name password access is possible. The latter values are indeed stored in `<clustername>.monitoring` secret and the url as present as an annotation on the same secret. But again please do not market this as `programmatic` access. Today we don't support that scenario.

Reply:
@nickytd do we have an issue or even a timeline for programmatic access? If we don't offer an alternative solution to users who need machine-to-machine access to e.g. Prometheus, we're forcing them to go with the deprecated option. I don't expect them to be happy with that.

Reply:
I am not aware of such plans. That is why we keep the current state preserving the use case.

Reply:
In that case I disagree with marking it as deprecated. It's a feature we're expected to provide, and without a replacement it should not be considered deprecated IMO.

Reply:
I will agree with you If you point me to the scenario description of `programatic access` service offering. Otherwise I am against this.

Reply:
> I will agree with you If you point me to the scenario description of programatic access service offering.

Was there a GEP discussing the move from basic auth to OIDC and dropping the support for federating metrics? I'm not entirely sure, where/if this kind of documentation exists at all for the observability area 🤔 @nickytd maybe you can help me out? Thanks!

Anyways, from the perspective of a Gardener consumer/user, basic auth access as an implementation of `programmatic access` is essential to federate metrics. If we are indeed deprecating it, a proper replacement needs to be in place before.

Reply:
To add to Hendrik's point: The ability to federate control plane metrics using the basic auth credentials has been an option available to users since before the OIDC ingresses were introduced.

Reply:
I see. Perhaps there is some historical service offering to which I am not aware. In this case, it is better, the service (scenario) owner of the `programatic access` to define to correct wording. 
As for the `OIDC` it  is an addition to the current auth & auth schemes and not a replacement.